connectors/trade-insight-sell-connector/src/main/resources/camel-connector-schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "component": {
+    "kind": "component",
+    "baseScheme": "rest",
+    "scheme": "trade-insight-sell",
+    "syntax": "trade-insight-sell:method:path:uriTemplate",
```

Comment:
I just blindly copied it from the existing [buy connector](https://github.com/redhat-ipaas/connectors/blob/master/connectors/trade-insight-buy-connector/src/main/resources/camel-connector-schema.json#L6)... it's unused in iPaaS so doesn't hurt to have it there.